layers/meta-opentrons/recipes-connectivity/networkmanager/files/NetworkManager-extra.conf

```diff
@@ -0,0 +1,2 @@
+[keyfile]
+path=/userfs/etc/NetworkManager/system-connections
```

Comment:
I'd love to keep the existence of `/userfs` hidden from most of these configurations, it feels a lot cleaner. Instead, let's use paths from where userfs is mounted - so let's make this `/var/lib/NetworkManager/system-connections`

Reply:
since this is a dropfile and its name doesn't matter, let's also give it a name like `system-connections-location.conf` or something